metadb/scrapers/spotgen.py

```diff
@@ -0,0 +1,32 @@
+import sys
+import spotipy
+
+
+sp = spotipy.Spotify()
+sp.trace = False
+
+def get_artist(name):
+	results = sp.search(q='artist:' + name, type='artist')
+	items = results['artists']['items']
+	if len(items) > 0:
+		return items[0]
+	else:
+		return None
+
+
+
+def scrape(mbid, artist, title, metadata=None):
+	data = None
+	spot_artist = get_artist(artist)
+	print('====', spot_artist['name'], '====')
+	if len(spot_artist['genres']) > 0:
```

Comment:
in the `get_artist` method you return `None` if there is no match. This code will not work if that is the case.